zeek/Reading-Zeek-Log-Formats.md

```diff
@@ -0,0 +1,158 @@
+# Reading Zeek Log Formats
+
+- [Summary](#summary)
+- [Zeek TSV](#zeek-tsv)
+- [Zeek NDJSON](#zeek-ndjson)
+- [The Role of `_path`](#the-role-of-_path)
+
+# Summary
+
+Zed is capable of reading multiple possible Zeek log formats. This document
```

Comment:
@nwt: Sure, happy to make this change. My prior vague wording was my way of acknowledging that there might be other niche log formats out there of which I'm unaware. But since we cover the two we know about here, I'm happy to make the wording more direct.